platform/linux-generic/include/odp/api/plat/packet_inlines.h

```diff
@@ -21,6 +21,9 @@
 /** @internal Inline function offsets */
 extern const _odp_packet_inline_offset_t _odp_packet_inline;
 
+/** @internal Pool inline function offsets */
+extern const _odp_pool_inline_offset_t _odp_pool_inline;
```

Comment:
This should go to pool_inlines.h

Reply:
pool_inlines.h would only include these two lines and looking at pool API there aren't any additional functions which should be inlined. Do you have a particular use case in mind for this header?

Reply:
Just wanted to have pool functions in pool header, etc.

Reply:
Since there are currently no functions in sight to put into pool_inlines.h I would suggest not to add a new header file. If/when some pool inline function are added later on the header can be created them. Is this OK for you?

Reply:
@matiaselo ok.

Reply:
The API being inlined is `odp_packet_pool()` so this is an artifact of the implementation of that API. So I'm OK with it being here since this is supposed to be fastpath code.